spec/requests/api_v1_work_spec.rb

```diff
@@ -26,100 +143,221 @@
-                                         "volume" => [],
+                                         "volume" => nil,
                                          "work_type" => "GenericWork",
                                          "workflow_status" => nil)
       end
 
       context 'with data when it exists' do
-        let(:work) { create(:work, visibility: 'open', abstract: abstract, creator: creator) }
-        let(:abstract) { 'Lorem ipsum dolor sit amet, consectetur adipiscing elit. Morbi at tincidunt nisl. Nulla et lacus consequat, interdum eros nec, pulvinar.' }
-        let(:creator) { ["[{\"creator_organization_name\":\"\",\"creator_given_name\":\"Bertie\",\"creator_family_name\":\"Wooles\",\"creator_name_type\":\"Personal\",\"creator_orcid\":\"0000 1111 2222 3333\",\"creator_isni\":\"\",\"creator_ror\":\"\",\"creator_grid\":\"\",\"creator_wikidata\":\"\"}]"] }
+        let(:work) { GenericWork.new(attributes) }
         it 'returns work json' do
+          work.save!
           get "/api/v1/tenant/#{account.tenant}/work/#{work.id}"
-          expect(json_response).to include("abstract" => abstract,
-                                           "creator" => [{ "creator_organization_name" => "", "creator_given_name" => "Bertie", "creator_family_name" => "Wooles", "creator_name_type" => "Personal", "creator_orcid" => "0000 1111 2222 3333", "creator_isni" => "", "creator_ror" => "", "creator_grid" => "", "creator_wikidata" => "" }])
+          expect(json_response).to include("abstract" => "Swedish comic about the adventures of the residents of Moominvalley.",
+                                           "adapted_from" => nil,
+                                           "additional_info" => ["Nothing to report"],
+                                           "additional_links" => nil,
+                                           "admin_set_name" => "",
+                                           "advisor" => nil,
+                                           "alternative_book_title" => nil,
+                                           "alternative_journal_title" => nil,
+                                           "alternative_title" => ["alt-title"],
+                                           "audience" => nil,
+                                           "book_title" => ["Book Title 1"],
+                                           "buy_book" => nil,
+                                           "challenged" => nil,
+                                           "citation" => nil,
+                                           "cname" => account.cname,
+                                           "collections" => [],
+                                           "committee_member" => nil,
+                                           "contributor" => [{ "contributor_family_name" => "Gnitset", "contributor_given_name" => "Jannet", "contributor_name_type" => "Personal", "contributor_orcid" => "https://orcid.org/0000-1234-5109-3702" }],
```

Comment:
Can we sort these lines out?

```
"contributor" => [{
    "contributor_family_name" => "Gnitset", 
    "contributor_given_name" => "Jannet", 
    "contributor_name_type" => "Personal", 
    "contributor_orcid" => "https://orcid.org/0000-1234-5109-3702"
 }]
```

Just helps loads with readability as longer lines are harder to parse.